app/controllers/consultees_controller.rb

```diff
@@ -3,12 +3,38 @@
 class ConsulteesController < AuthenticationController
   before_action :set_planning_application
 
+  def edit
+    @consultee = planning_application.consultees.find(params[:id])
```

Comment:
If `params[:id]` is an integer we should use `Integer(params[:id])` to ensure that's the case otherwise it may be possible to inject hash arguments to `find`.

Reply:
Doesn't look like `find` takes hash arguments, but it does take array arguments and also string arguments, and string arguments could either be IDs in string format or arbitrary SQL conditions 😱, so… your point stands.